docs/Audio.md

```diff
@@ -29,12 +29,62 @@ Determines if a referenced track is currently playing. Requires a valid referenc
 
 Ends the currently playing BGM. Will restart if resumed.
 
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.)
+
 ### Suspend BGM
 
 Suspends the currently playing BGM. If resumed, will continue from the time it was suspended.
 
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.)
+
 ### Resume BGM
 
+Will resume the music where it left off, with a fade in. The fade duration is set in the BGM controller's parameters.
+
+Also exists with an "Optional Fade" variant, which requires a fade duration (in sec.) which will override the BGM controller's.
+
+### Get Musical Time
+
+Given a BGM controller, gets the play time of its currently BGM track. The play time's value ("Musical time") is not in seconds, and does not directly scale with real-time values.
```

Comment:
Out of curiosity, what scale is this value in? Game frames/ticks? Or do we not know?

Reply:
It's a bit complex, I'll refer to a section for that!